git_machete/github.py

```diff
@@ -21,12 +23,63 @@ def __repr__(self) -> str:
         return f"PR #{self.number} by {self.user}: {self.head} -> {self.base}"
 
 
-GITHUB_TOKEN_ENV_VAR = 'GITHUB_TOKEN'
+GITHUB_TOKEN_ENV_VAR = "GITHUB_TOKEN"
```

Comment:
`black`, right? 🏴

Reply:
Pls remind me... `black` applies the style that's strictly **more opinionated** than pep8 right? So `black`-formatted sources will always conform to pep8?

Reply:
Yes, exactly. `black` is an delightfully opinionated superset of the pep8 format guidelines.
One of it's ...more controversial opinions... is the normalization of string quote formats, but...
"It's opinionated so you don't have to be."

I'll back this out now, would still be in favor of `black`-ening at some point after your `hub` PR lands.